_docs/quickstart.md

```diff
@@ -230,8 +234,8 @@ For example, the count of total Cores available on rank 0 can be obtained
 from the kvs via:
 
 ```console
-$ flux kvs get resource.hwloc.by_rank.0.Core
-16
+$ flux kvs get resource.hwloc.by_rank
+{"[0-2]": {"NUMANode": 2, "Package": 2, "Core": 36, "PU": 36, "cpuset": "0-35"}}
```

Comment:
again, only 3 rank's info was found by hwloc, but all other examples above (besides hwloc info) note an instance size of 4? Did something go wrong in the example instance?